src/hub.html

```diff
@@ -55,31 +55,20 @@
     <div id="support-root"></div>
 
     <a-scene
-        disable-resize
-        embedded
         vr-mode-ui="enabled: false"
         loading-screen="enabled: false"
         hubs-systems
         capture-system
         listed-media
         local-audio-analyser
         class="grab-cursor"
-        renderer="
-            antialias: true;
-            colorManagement: true;
-            sortObjects: true;
-            physicallyCorrectLights: true;
-            alpha: false;
-            webgl2: true;
-            multiview: false;"
```

Comment:
Is it ok to remove renderer parameters? Some custom client devs might need to control them, for example turning off anti-aliasing for better performance. Or we will think how we allow them to control the renderer parameters later?

Reply:
We now create the renderer directly in app.js so they can modify these properties themselves, no hoops to jump through https://github.com/mozilla/hubs/pull/5318/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R35